.github/workflows/translations.yml

```diff
@@ -0,0 +1,72 @@
+name: translation-workflow
+run-name: regenerating pot file
+
+on:
+  workflow_run:
+    workflows: ["Release"]
+    types: [requested]
+#  push:
+#    branches:
+#      - master
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  update-pot-file:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout master branch
+        uses: actions/checkout@v3
+
+      - name: Install PHP
+        uses: shivammathur/setup-php@master
+        with:
+          php-version: "7.4"
+
+      - name: Setup WP-CLI
+        uses: godaddy-wordpress/setup-wp-cli@1
+
+      #######################
+      ### MAKE EDITS HERE ###
+      #######################
+      # edit .pot location
+      # remove/edit --subtract option
+      # edit --domain option
+      # add any folders for excluding if necessary in --exclude option
+      #######################
+      - name: Generate new pot file
+        run: >
+              wp i18n make-pot . languages/default.pot  --skip-audit --subtract="languages/terms_to_exclude.pot"
+              --domain="disciple-tools-prayer-campaigns"
+              --exclude=""
+      #######################
+
+      - name: Check diff output
+        run: git diff | wc -l
+
+      - name: Add changes
+        run: git add -A
+
+      - name: Check status
+        run: git status
+
+      #######################
+      ### MAKE EDITS HERE ###
+      #######################
+      # edit name and email
+      #######################
+      - name: push changes
+        uses: github-actions-x/commit@master
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          push-branch: master
+          commit-message: 'update pot translation file'
+          name: squigglybob
+          email: nathinabob@gmail.com
```

Comment:
wip?

Reply:
it needs a valid name and email for the commit to work i found, so needs to be changed to whatever name and email it should be for this repo

Reply:
changed to use the current github actor